src/object_sync/hello.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include <nan.h>
+
+namespace object_sync {
+
+    /**
+     * HelloObject class
+     * This is in a header file so we can access it across other .cpp files if necessary
+     * Also, this class adheres to the rule of Zero because we define no custom destructor or copy constructor
+     */
+    class HelloObject: public Nan::ObjectWrap {
+
+        public:
+            // initializer
+            static void Init(v8::Local<v8::Object> target);
+
+            // methods required for the constructor
+            static NAN_METHOD(New);
+            //static Nan::Persistent<v8::Function> &constructor();
```

Comment:
@springmeyer Any need to add a special constructor for this sync Object example?

Somewhat related, I'm debating what to remove from `src/hello_world.cpp` and add to the new examples.